python/llm/src/ipex_llm/transformers/low_bit_linear.py

```diff
@@ -405,6 +405,7 @@ def use_batch_forward(x: torch.Tensor, qtype: int, output_len: int):
             or (device in ["arc", "flex"] and qtype in [SYM_INT8, FP4])
             or (device in ["arc", "flex", "mtl"] and qtype in [FP8E4])
             or (device in ["lnl"] and qtype in [SYM_INT4] and x.shape[1] % 512 == 0)
+            or (device in ["bmg"] and qtype in [SYM_INT4])
```

Comment:
To support more inference and serving requirements on BMG, shall we also verify that FP8/INT8 can benefit from use_batch_forward. @jason-dai

Reply:
I collected FP8 data on BMG using the all-in-one. Models larger than 7B cause the machine to crash, while models smaller than 7B show a 1-2% improvement in next-token performance after enabling the batch kernel. 

Should we enable the batch kernel for qtype=FP8E5 as well?